# The directory name to which we want to save the two point clouds
string dirname
---
# Number of points in the cloud
bool success

